cf-check/dump.c

```diff
@@ -154,23 +156,27 @@ static void print_struct_averages(
     else
     {
         // TODO: clean up Averages
```

Comment:
@olehermanse Wasn't sure what "clean up Averages" meant? Maybe adding ts_key handling is this cleanup?